src/main/java/views/session_load/SessionLoadC.java

```diff
@@ -0,0 +1,41 @@
+package views.session_load;
+
+import use_cases.session_load.SessionLoadException;
+import use_cases.session_load.SessionLoadIB;
+import use_cases.session_load.SessionLoadID;
+import use_cases.session_load.SessionLoadOD;
+
+public class SessionLoadC {
+    private final SessionLoadIB interactor;
+
+    public SessionLoadC(SessionLoadIB interactor) {
+        this.interactor = interactor;
+    }
+
+    /**
+     * Calls the interactor to load a new session from a file
+     *
+     * @param filename the filename to load the session from
+     * @return a SessionLoadOD object with the output data to be used by the View
+     * @throws SessionLoadException if there is an error while opening the file
+     */
+    public SessionLoadOD loadFile(String filename) throws SessionLoadException {
+        SessionLoadID inputData = new SessionLoadID(filename);
+        return this.interactor.load(inputData);
+    }
+
+    /**
+     * Calls the interactor to create a new empty session
+     *
+     * @return a SessionLoadOD object with the output data to be used by the View
+     */
+    public SessionLoadOD loadNew() {
+        SessionLoadID inputData = new SessionLoadID();
+        try {
+            return this.interactor.load(inputData);
+        } catch (SessionLoadException e) {
+            // Loading a new empty session should never fail
+            throw new RuntimeException();
```

Comment:
I think I mentioned it above but I am a little confused on why you throw RuntimeException here instead of handling it and printing a message

Reply:
I didn't want to make the loadNew method throw SessionLoadException, because creating a new session is just creating a new SessionStorage object and that has no way to fail. But because the interactor uses the same load method for both an empty session (which never throws) or loading from a file (which may throw), the compiler forces me to handle the exception. 

So, the inside of the catch should never run at all no matter what, and if it ever *does* run, it means the program is in such a screwed up state that we somehow couldn't even create a SessionStorage object, and in that case the program should just crash because there's nothing else to do. Maybe changing it to an assertion instead of just throwing an exception could communicate the intent better?

Reply:
Hmmm I think what you said makes sense. Its up to you if you want to switch to an Assertion, but you're explanation makes sense to me. Maybe you could include a brief explanation in the method Javadoc?